### ISSUE TITLE
Keras Models with tf.sparse.sparse_dense_matmul can't be saved - Not JSON Serializable 

### ISSUE THREAD

**Issue by Asorie** [tensorflow/tensorflow#31607]
**System information**
- OS Platform and Distribution: Windows 10 Version 1607
- TensorFlow version: 2.0.0-beta1
- Python version: 3.7.3

**Describe the current behavior**
Error:
```
Exception has occurred: TypeError
('Not JSON Serializable:', b'\n\ttranspose\x12\tTranspose\x1a\x10dense_1/Identity\x1a\x0etranspose/perm*\x0b\n\x05Tperm\x12\x020\x03*\x07\n\x01T\x12\x020\x01')
  File "C:\Visual_Studio_Codes\Sparse_Save.py", line 30, in <module>
    model.save("model.h5")
```

**Describe the expected behavior**
Save without error

**Code to reproduce the issue**
```
import tensorflow as tf
from tensorflow.keras.layers import (Dense, Input, Lambda)
from tensorflow.keras.models import Model, Sequential
from scipy import sparse
import numpy as np


def layer_lambda(input_x):
    sparse = input_x[0]
    dense = input_x[1]
    dense = tf.transpose(dense)
    y = tf.sparse.sparse_dense_matmul(sparse, dense)
    return tf.transpose(y)


dense_mat = np.eye(30, 30, dtype=np.float32)
sparse_mat = sparse.coo_matrix(dense_mat)
sparse_indices = np.mat([sparse_mat.row, sparse_mat.col]).transpose()
sparse_tensor = tf.SparseTensor(sparse_indices, sparse_mat.data, sparse_mat.shape)

model = Sequential()
model_input = Input(shape=(20,))
x = Dense(20)(model_input)
x = Dense(30)(x)
x = Lambda(layer_lambda, output_shape=(None, 30, 30))([sparse_tensor, x])
model = Model(model_input, x)

model.predict([[np.ones(20)]])

model.save("model.h5")
```

**Comment by gowthamkpr** [comment]
@Asorie Can you please take a look at the similar [issue](https://github.com/tensorflow/tensorflow/issues/27112) and let me know if it helps. Thanks!

**Comment by ymodak** [comment]
This is fixed with latest version of TF-2.0 nightly build. Thanks!

**Comment by Asorie** [comment]
All in [this issue](https://github.com/tensorflow/tensorflow/issues/27112) suggested solutions dont work in this case. 
I ended up using `pip install tf-nightly-gpu`. This solved this issue
